Makefile.am

```diff
@@ -206,7 +207,12 @@ endif
 if FUNCTIONAL_TESTS
 CHECK_DEPS += functional-tests
 functional-tests: cc-oci-runtime
+	$(AM_V_GEN)test -n "$(BUNDLE_TEST_PATH)" && \
+		echo "Using bundle '$(BUNDLE_TEST_PATH)'" || true
 	@$(BATS_PATH) $(srcdir)/tests/functional/
+	$(AM_V_GEN)test -e "$(BUNDLE_TEST_PATH)" -a "$(REMOVE_BUNDLE_ON_SUCCESS)" = "yes" \
```

Comment:
Random comment (nothing to do here): I think you're over doing it :) removing the bundle on "make clean" would have been enough?